wdfn-server/waterdata/templates/partials/footer.html

```diff
@@ -1,10 +1,12 @@
-<div class="user-feedback">
-    <a href="{{ url_for('questions_comments', email_for_data_questions=email_for_data_questions) }}"
-       target="_blank"
-       ga-on="click" ga-event-category="navigation" ga-event-action="toFeedBackForm_floatingLink">
-        <p class="usa-prose">Questions or Comments</p>
-    </a>
-</div>
+{% if request.endpoint == 'monitoring_location' or request.endpoint == 'networks'%}
```

Comment:
This is fine for now. I think if we begin to add other pages, we may want to implement a context processor on the flask side to insert a variable that enables/disables the feedback section. But this works so let's leave it.

Reply:
Sounds good. Reading about Context Processors right now . . .

Reply:
it's just a way to pass a variable to any page. We used a context processor to pass whether a user was authenticated: https://github.com/usgs/pubswh-ui/blob/5ede5634451681f1584885da49d46d6a77803fd7/server/pubs_ui/auth/views.py#L58

Reply:
Okay, I think I got the context processor working correctly. A pretty handy feature.

Reply:
Yes it is.Bonus is that you can write tests for it too. I didn't expect that you would do this but if you added it, it should really have tests

Reply:
Will do.